### DISCUSSION TITLE
Osu lazer android input keeps pressed after release.

### DISCUSSION THREAD

**Discussion by Robiimadot** [ppy/osu#22577] [Q&A]
### Type

Game behaviour

### Bug description

When played harder 7k maps i realized that the inputs are pressed even after i stoped pressing the buttons.

(And i don't know why the game was stopped at the beginning as well, (but it helped me record this video so I'm not complaining), might be that i started recording the video)

### Screenshots or videos

https://user-images.githubusercontent.com/65800580/217621166-5a190b11-8311-4b6b-8351-11b962a469a8.mp4

### Version

2023.207.0-lazer

### Logs

[runtime.log](https://github.com/ppy/osu/files/10688830/runtime.log)
[network.log](https://github.com/ppy/osu/files/10688834/network.log)
[performance.log](https://github.com/ppy/osu/files/10688824/performance.log)

**Comment by Robiimadot** [comment]
Phone: huawei p30

**Comment by peppy** [comment]
Hard to say without more details, but this looks like it may be out of our control (an issue with your devices USB stack or something else).

Are you able to record the same video after bringing up the frame time displays (ctrl+f11 twice)?

**Reply by Robiimadot** [reply]
I tried it, but it doesn't bring up the debug menu, it just makes screenshots with ctrl + f11.

**Reply by peppy** [reply]
If you mean the game screenshot functionality, that's F12. If you mean a system-level screenshot then... I'm not sure how to get around that.

**Reply by Robiimadot** [reply]
I think it's system level screenshot
